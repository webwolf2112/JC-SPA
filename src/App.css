.App {
  text-align: center;
  color: '#FFF';
  font-family: 'Roboto', sans-serif;
}


.App-header {
  background-color: #282c34;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

button {
  font-family: 'Roboto', sans-serif;
  background-color: #2cc692; 
  font-size: medium;
  border: 1px solid #a9a9a9;
}

button.inverse {
  background-color: #fefefe;
  border: 1px solid #2cc692;
}

button:disabled {
  background-color: #ccc;
  border: 1px solid #a9a9a9;
}

ul {
  list-style-type: none;
}