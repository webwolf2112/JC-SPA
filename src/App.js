import logo from './logo.svg';
import './App.css';
import UsersContainer from './users-container';

function App() {
  return (
    <div className="App">
      <header className="App-header">
          Header Stuff here
      </header>
      <UsersContainer />
    </div>
  );
}

export default App;
