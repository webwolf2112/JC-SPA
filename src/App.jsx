import './App.css';
import UsersContainer from './users-container';

function App() {
  return (
    <div className="App">
      <UsersContainer />
    </div>
  );
}

export default App;
