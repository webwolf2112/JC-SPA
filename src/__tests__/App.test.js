import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

test('Link changes the class when hovered', () => {
  const wrapper = shallow(
    <App/>,
  );

  expect(wrapper).toMatchSnapshot();
});