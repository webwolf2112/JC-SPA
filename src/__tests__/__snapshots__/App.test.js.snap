// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App It should render correctly 1`] = `
<div
  className="App"
>
  <UsersContainer />
</div>
`;
